ci/vars-functional-test.yml

```diff
@@ -9,7 +9,11 @@ pre_tests_00_run_functional_test:
   source: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}/ci/run_cloudops_tests_osp18.yml"
   type: playbook
   config_file: "{{ ansible_user_dir }}/{{ zuul.projects['github.com/infrawatch/feature-verification-tests'].src_dir }}/ci/ansible.cfg"
-
+pre_tests_99_collect_results:
```

Comment:
Is there a reason why the hook is called differently here and is called in a different stage than in all the other vars* files?